test/new-e2e/pkg/utils/e2e/client/docker.go

```diff
@@ -103,19 +105,41 @@ func (docker *Docker) ExecuteCommandStdoutStdErr(containerName string, commands
 	execInspectResp, err := docker.client.ContainerExecInspect(context, execCreateResp.ID)
 	require.NoError(docker.t, err)
 
-	output := outBuf.String()
-	errOutput := errBuf.String()
+	stdout = outBuf.String()
+	stderr = errBuf.String()
 
 	if execInspectResp.ExitCode != 0 {
-		return "", "", fmt.Errorf("error when running command %v on container %v: %v %v", commands, containerName, output, errOutput)
+		return "", "", fmt.Errorf("error when running command %v on container %v, containerID %v:\n   exit code: %d\n   stdout: %v\n   stderr: %v", commands, containerName, "vvvvvv", execInspectResp.ExitCode, stdout, stderr)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** 

Why do we need to log the `containerID` if it will always be `vvvvvv`? I'm not sure what `vvvvvv` means here in fact 😅 ‏

Reply:
Oh thank you, good catch! This is a leftover. I believed that this method was broken as it was expecting the `containerID`, so I implemented the logic to find a container id from its name. Plot twist, after discussing with @ogaca-dd that implemented first the feature, I had the doubt it was actually working in the first place. And it was, so removed the containerID part, left the `ListContainers` as it can be still be helpful